src/landing-page/landing-page.js

```diff
@@ -20,55 +20,55 @@ export default function LandingPage() {
 
 	return (
 
-		<VStack alignment="center">
+		<>
 			{ createModalType && (
 				<CreateThemeModal
 					creationType={ createModalType }
 					onRequestClose={ () => setCreateModalType( false ) }
 				/>
 			) }
-			<img src="/wp-content/plugins/create-block-theme/.wordpress-org/banner-772x250.png" alt="Create Block Theme Logo" />
 
-			<VStack alignment="left" style={{ maxWidth: "700px"}}>
-			<p>
-				Create Block Theme is a tool to help you make Block Themes using the WordPress Editor.
-				It adds tools to the Editor to help you create and manage your theme.
-				Themes created with Create Block Theme are built using the WordPress Editor and are
-				compatible with the Full Site Editing features in WordPress.  Themes created with Create Block Theme
-				don't require Create Block Theme to be installed on the site where the theme is used.
-			</p>
+		<div style={{width:"100%", backgroundColor: "#2D59F2", marginLeft: "-20px", paddingRight: "20px" }}>
+			<img src="/wp-content/plugins/create-block-theme/assets/header_logo.png" alt="Create Block Theme Logo" />
+		</div>
+
+		<HStack alignment="topLeft" style={{paddingTop: "40px"}}>
+
+			<VStack alignment="left" style={{ maxWidth: "620px", padding: "0 40px"}}>
+				<h1>What would you like to do?</h1>
+				<p>You can do everything from within the Editor but here are a few things you can do to get started.</p>
 
-			<p><strong>What would you like to do?</strong><br/>
-			You can do everything from within the Editor but here are a few things you can do to get started.</p>
 
-			<HStack alignment="left">
-				<Button variant="secondary">Export this theme as a Zip File.</Button>
+				<Button variant="link" style={{ fontSize:"22px"}}>Export this theme as a Zip File.</Button>
 				<p>Export a zip file ready to be imported into another WordPress environment.</p>
-			</HStack>
 
-			<HStack alignment="left">
-				<Button variant="secondary" onClick={()=>setCreateModalType('blank')}>Create a new Blank Theme</Button>
+				<Button variant="link" style={{ fontSize:"22px"}} onClick={()=>setCreateModalType('blank')}>Create a new Blank Theme</Button>
 				<p>Start from scratch!  Create a blank theme to get started with your own design ideas.</p>
-			</HStack>
 
-			<HStack alignment="left">
-				<Button variant="secondary" onClick={()=>setCreateModalType('clone')}>Create a Clone of This Theme</Button><br/>
+				<Button variant="link" style={{ fontSize:"22px"}} onClick={()=>setCreateModalType('clone')}>Create a Clone of This Theme</Button><br/>
```

Comment:
```suggestion
				<Button variant="link" style={{ fontSize:"22px"}} onClick={()=>setCreateModalType('clone')}>Create a Clone of This Theme</Button>
```

Reply:
Sorry, had to clobber your change.  Had quite a bit of styling work to push up.

Reply:
I don't know what clobber means 😂
But it's fine if it's gone :)